scripts/test_twemoji_mapping.py

```diff
@@ -54,16 +54,22 @@
 
     n = len(mapping)
     for i, emoji in enumerate(mapping, start=1):
-        emoji_surrogates = emoji["surrogates"]
-        name = emoji["primaryName"]
-        emoji = emojis.UnicodeEmoji.parse(emoji_surrogates)
-
-        if emoji.filename in known_files:
-            valid_emojis.append((emoji_surrogates, name))
-            print("[  OK  ]", f"{i}/{n}", name, *map(hex, map(ord, emoji_surrogates)), emoji.url)
-        else:
+        try:
+            emoji_surrogates = emoji["surrogates"]
+            name = emoji["primaryName"]
+            emoji = emojis.UnicodeEmoji.parse(emoji_surrogates)
+
+            if emoji.filename in known_files:
+                valid_emojis.append((emoji_surrogates, name))
+                print("[  OK  ]", f"{i}/{n}", name, *map(hex, map(ord, emoji_surrogates)), emoji.url)
+            else:
+                invalid_emojis.append((emoji_surrogates, name))
```

Comment:
Also if we kept the whole parse validates stuff behaviour around couldn't this pipeline in theory just directly call UnicodeEmoji's init instead of the parse method to get the same result?